src/IntegratedExperts/BehatScreenshotExtension/ServiceContainer/BehatScreenshotExtension.php

```diff
@@ -55,7 +55,9 @@ public function configure(ArrayNodeDefinition $builder)
         $builder->children()
             ->scalarNode('dir')->isRequired()->cannotBeEmpty()->end()
             ->scalarNode('fail')->isRequired()->cannotBeEmpty()->end()
-            ->scalarNode('purge')->isRequired()->cannotBeEmpty()->end();
+            ->scalarNode('purge')->isRequired()->cannotBeEmpty()->end()
+            ->scalarNode('png')->defaultTrue()->end()
+            ->scalarNode('html')->defaultFalse()->end();
```

Comment:
options should use `booleanNode`

I'm aware tat `purge` uses `scalarNode` which is incorrect and needs to be fixed.